### PR TITLE
DOC: adds more information about transforms

### PR DESCRIPTION
**Pull request by deniederhut**
And a rough equivalency table between libraries.

### CODE REVIEW

File: papers/joss/paper.md [hunk 1]
```diff
@@ -21,9 +21,12 @@ bibliography: paper.bib
...
+In the domain of NLP, there is an analogous set of transformations that have been shown to increase model accuracy. Common transformations include adding superfluous whitespace, swapping an individual word for a synonym, replacing a word with a misspelled version, and round-tripping a sentence through a translation model into another language and back again [@zhang2015characterlevel;@coulombe2018text]. Some transforms, like misspellings or synonyms, attempt to directly model an unobserved but common change in the input features. Others, like adding a space in the middle of a word, act similarly to dropout for word-level models, in that they effectively the word as an input (e.g. the two halves of the word are unlikely to be present in the vocabulary).
```
Comment (sara-02) [inline papers/joss/paper.md]: Missing a word after "...effectively the word..."

File: papers/joss/paper.md [hunk 2]
```diff
@@ -21,9 +21,12 @@ bibliography: paper.bib
...
+
+There are also sets of transforms whose aim is to make models more robust in the face of adversarial inputs. These transforms include replacing letters with symbols that are visually similar, adding nuisance characters around words, adding random characters at the end of the document, and appending the word "love" to the end of a sentence [@grndahl2018need]. Adding nuisance characters around words, or replacing characters with symbols, disrupt the bahvior of vocabulary-based models, in that they will not know how to encode an input like "sch00l", although it is visually clear to an experienced reader of English that this is the word "school". Adding words with very positive or very negative connotations have been shown to be an effective way to disrupt sentiment classifiers, which often rely on counting the number of times such words appear.
```
Comment (sara-02) [inline papers/joss/paper.md]: Typo in `bahvior`, or was it intentional ;)

File: papers/joss/paper.md [hunk 3]
```diff
@@ -56,6 +59,29 @@ and substituting numbers and symbols for letters they resemble:
...
 
+There are a small number of libraries that contain text augmentation functions, for example [noisemix](https://github.com/noisemix/noisemix) and [EDA](https://github.com/jasonwei20/eda_nlp) [@wei2019eda]. However, many of these are published only as a part of a conference paper, and are not actively maintained. EDA, for example, contains the warning "The code is not documented". They also tend to contain a subset of the common kinds of text transformations. EDA contains four functions, two of which are varities of synonym replacement. Noisemix is more feature complete, and contains a function that adds typos to words. There is not an exact one-to-one correspondence between all the capabilities of each library, but here is a table with a rough alignment:
+
+
+transformation       | niacin | noisemix | eda
```
Comment (sara-02) [inline papers/joss/paper.md]: +1

File: papers/joss/paper.md (hunk 1 above)
Comment (deniederhut) [reply papers/joss/paper.md]: 🤦‍♀️ I accidentally the words a lot

File: papers/joss/paper.md (hunk 2 above)
Comment (deniederhut) [reply papers/joss/paper.md]: Lol it might feel a bit bah

**Pull request merged**